e2e/Makefile

```diff
@@ -1,37 +1,48 @@
 include ../Makefile.build.mk
 
 BIN_DIR := $(shell pwd)/bin
+KOTS_BIN_DIR := $(shell dirname $(shell pwd))/bin
 
 SHELL := /bin/bash
-PATH := $(BIN_DIR):$(shell npm bin --prefix=$(BIN_DIR)):$(shell dirname $(shell pwd))/bin:$(PATH)
 
 .PHONY: all
-all: build test
+all: build deps test
 
 .PHONY: deps
-deps: export INSTALL_DIR=$(BIN_DIR)
 deps:
-	./hack/deps.sh
+	docker build -t e2e-deps -f ./hack/Dockerfile ./hack
 
 .PHONY: build
 build:
 	go test $(BUILDFLAGS) -c -o bin/e2e.test .
 
 .PHONY: test
 test: export GINKGO_EDITOR_INTEGRATION=1 # disable error on programatic focus
-test: KOTSADM_IMAGE_TAG ?= v0.0.0-nightly
+test: KOTSADM_IMAGE_TAG ?= alpha
 test: TESTIM_BRANCH ?= master
 test: SKIP_TEARDOWN ?= 0
+ifneq ($(EXISTING_KUBECONFIG),)
+test: EXISTING_KUBECONFIG_VOLUME_MOUNT := "-v=$(EXISTING_KUBECONFIG):$(EXISTING_KUBECONFIG)"
+endif
 test:
-	e2e.test \
-		-test.v \
-		--ginkgo.focus="$(FOCUS)" \
-		--testim-branch=$(TESTIM_BRANCH) \
-		--existing-kubeconfig=$(EXISTING_KUBECONFIG) \
-		--kotsadm-image-registry=$(KOTSADM_IMAGE_REGISTRY) \
-		--kotsadm-image-namespace=$(KOTSADM_IMAGE_NAMESPACE) \
-		--kotsadm-image-tag=$(KOTSADM_IMAGE_TAG) \
-		--skip-teardown=$(SKIP_TEARDOWN)
+	export TESTIM_ACCESS_TOKEN
```

Comment:
what is this always empty?

Reply:
It's just exporting a local shell variable for use in the docker command